src/cc/common.cc

```diff
@@ -48,4 +49,16 @@ std::vector<int> get_possible_cpus() {
   return read_cpu_range("/sys/devices/system/cpu/possible");
 }
 
+std::string get_pid_exe(pid_t pid) {
+  char exe_path[4096];
```

Comment:
We know how big a pid can be, so the stack size 4096 is too big. Maximumly 64 or 128 should be enough.

Reply:
This is path of the executable binary so PATH_MAX is expected. The `procfs` link path is `exe_link`:)

Reply:
Sorry, my mistake.